src/components/mgmt/DiningManagement.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+    <div class="dining-management">
+        <CustomHeader subtitle="Dining" />
+        <NavBar />
+        <h2>Dining Management</h2>
+        <div class="main-container">
+            <div class="content">
+                <div class="col left">
+                    <ReservationTable 
+                        id="spa-res"
+                        :columns="resCols"
+                        :rowData="reservations"
+                        title="Reservations"
+                        @updateItem="updateReservation"
+                        @deleteItem="deleteReservation"
+                    />
+                </div>
+                <div class="services">
+                    <div class="title">
+                        <h3>Key Stats</h3>
+                    </div>
+                    <h4>Today's Dinner</h4>
+                    <div v-for="(item, index) in items" :key="index" class="event-item">
+                        <h5 class="event-title">{{ item.title }}</h5>
+                        <p class="event-list"><span>{{ item.number }}</span></p>
+                    </div>
+                    <Button class="btn">Update Menu</Button>
```

Comment:
Move the Button element to below the closing \</div> tag that is now on line 28. You'll then need to wrap the "services" div and the Button in another div: \<div class="col right"> in order to get the layout to display properly.